source/documentation/getting-started/kubectl-config.html.md.erb

```diff
@@ -37,12 +37,10 @@ You will need to:
 
 ### Joining the GitHub organisation
 
-You can request access to the `ministryofjustice` GitHub organisation by:
+If you have a `@digital.justice.gov.uk` or `@justice.gov.uk` email address and only require access to the `ministryofjustice` Organisation, you can simply join via [SSO](https://github.com/orgs/ministryofjustice/sso) to join.
```

Comment:
doesn't this read you can simply join via SSO to join? With two "to join"s? Sounds funny no?